src/pymmcore/__init__.pyi

```diff
@@ -7,12 +7,14 @@
 # autogenerating "good" type hints. (along with a script that might help for anyone
 # so inclined)
 from __future__ import annotations
-from typing import Any, Final, List, Literal, overload, Sequence, Tuple, Union
-from typing_extensions import deprecated
+from typing import Any, Final, List, Literal, NewType, overload, Sequence, Tuple, Union
+from typing_extensions import TypeAlias
```

Comment:
`TypeAlias` no longer used? (Sorry, we should have Ruff set up.)

Reply:
good catch